### DISCUSSION TITLE
Unable to launch Terminal app

### DISCUSSION THREAD

**Discussion by neolib** [microsoft/terminal#14679] [General]
I've got a weird specific problem. I am not able to launch Terminal app in a VirtualBox VM.

Event log:
`0x80070005: Cannot create the Desktop AppX container for package Microsoft.WindowsTerminal_1.15.3465.0_x64__8wekyb3d8bbwe because an error was encountered configuring the runtime.`

However, I can install and launch Terminal if I create a new user account (in the same VM).

EDIT:
The VM has the latest monthly update (Jan 2023); also has Visual Studio 2019 installed.

**Comment by OS-Scratch** [comment]
Try Visual Studio 2022 17.12 and later with the required workloads (can be found in the README). The problem also might be in the VM’s configurations. It also might be that you haven’t enabled developer mode. Also, your acc might not have admin permissions.

Hope one of these solutions help.

**Comment by zadjii-msft** [comment]
@neolib Can you try a newer version of the Terminal package? Without digging into it more, I idly wonder if this is one of the VCRT or MUX dependency bugs that we resolved sometime in the last year. 1.15 is a pretty old version of the Terminal 😅
